main.py

```diff
@@ -1,7 +1,68 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
```

Comment:
Check p2-p3 of cheklist. Use for loops to call create methods within to avoid repetition